openant/devices/shift.py

```diff
@@ -43,6 +66,25 @@ class ShiftData(DeviceData):
     invalid_outboard_front: int = field(default=0)
     shift_failure_rear: int = field(default=0)
     shift_failure_front: int = field(default=0)
+    function_set_1_configuration: FunctionSetConfiguration = field(default=FunctionSetConfiguration())
```

Comment:
The tests are failing because a mutable default needs to use `default_factory` here (and all `FunctionSetConfiguration()`.

Reply:
Updated to use default_factory